blockchain/tx_pool.go

```diff
@@ -870,7 +886,7 @@ func (pool *TxPool) add(tx *types.Transaction, local bool) (bool, error) {
 	from, _ := types.Sender(pool.signer, tx) // already validated
 	if list := pool.pending[from]; list != nil && list.Overlaps(tx) {
 		// Nonce already pending, check if required price bump is met
-		inserted, old := list.Add(tx, pool.config.PriceBump)
+		inserted, old := list.Add(tx, pool.config.PriceBump, pool.chainconfig.IsKIP71ForkEnabled(pool.chain.CurrentBlock().Number()))
```

Comment:
Is this fine to use current block number? not current block number + 1?

Reply:
Yes, you are correct. Thanks!